### PR TITLE
Back to dev.

### PR DESCRIPTION
**Pull request by sherakama**

### CODE REVIEW

**Comment by sherakama** [comment]
Back to dev.

File: stanford_gallery.info [hunk 1]
```diff
@@ -2,7 +2,7 @@ name = Stanford Gallery
 description = A responsive Image Gallery Feature
 core = 7.x
 package = Stanford
-version = 7.x-2.3
+version = 7.x-2.4-dev
```
Comment (pookmish) [inline stanford_gallery.info:5]: aren't we calling it `7.x-2.x`?

**Review by pookmish — CHANGES_REQUESTED** [review]

**Comment by sherakama** [comment]
We have, and were, and do use that in a number of our modules but it is not really what we were supposed to be doing

https://www.drupal.org/docs/7/choosing-drupal-version/version-numbers

The branch name is 7.x-x.x but the module version number is 7.x-#.#-suffix

This only gets worse in D8.

**Comment by pookmish** [comment]
Works for me

**Pull request merged**